### PR TITLE
Update remove-a-domain.md

### PR DESCRIPTION
**Pull request by msbemba**
Added "You get an error message when you try to remove a domain from Office 365 or if this is Still not working? Your domain might need to be manually removed."

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/2570

File: microsoft-365/admin/get-help-with-domains/remove-a-domain.md [hunk 1]
```diff
@@ -216,8 +216,8 @@ If you have hundreds or thousands of users, use PowerShell to query for all user
 
 > [!NOTE]
 > You can't remove the [".onmicrosoft.com"](https://docs.microsoft.com/microsoft-365/admin/setup/domains-faq) domain from your account.
-  
-Still not working? Your domain might need to be manually removed. [Give us a call](../contact-support-for-business-products.md) and we'll help you take care of it!
+
+You get an error message when you try to remove a domain from Office 365 or if this is Still not working? Your domain might need to be manually removed. [Give us a call](../contact-support-for-business-products.md) and we'll help you take care of it!
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/get-help-with-domains/remove-a-domain.md]:
```suggestion
Getting an error message when you try to remove a domain from Office 365 or this is still not working? Your domain might need to be manually removed. [Give us a call](../contact-support-for-business-products.md) and we'll help you take care of it!
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@pebaum Please merge this PR. Thanks.

**Comment by yogkumgit** [comment]
@pebaum Gentle reminder, please merge this PR. Thanks

**Comment by pebaum** [comment]
this edit doesn't follow product naming guidelines.

**Pull request closed without merging**